modules/javafx.base/src/main/java/javafx/util/Subscription.java

```diff
@@ -29,13 +29,24 @@
 import java.util.Objects;
 
 /**
- * A {@code Subscription} represents the encapsulation of the result of an action
- * that can be cancelled. Subscriptions can be obtained, for example, as a result
+ * Represents a cancel or cleanup operation for an action be cancelled or
+ * allocated resources. Subscriptions can be obtained, for example, as a result
```

Comment:
Somethign is missing in the sentence, should it be "...for an action that can be cancelled or that allocates resources"

Reply:
Sorry, missed that somehow, fixed.